### PR TITLE
Antrea-proxy

### PR DESCRIPTION
**Pull request by knabben**
```
❯ kubectl get pods                                                                                                                                                                                                                                                   
Unable to connect to the server: net/http: TLS handshake timeout
```
```
2021-12-22T01:42:03.787880416Z stderr F E1222 01:42:03.787694       1 available_controller.go:524] v1beta1.system.antrea.tanzu.vmware.com failed with: failing or missing response from https://10.96.128.183:443/apis/system.antrea.tanzu.vmware.com/v1beta1: Get "https://10.96.128.183:443/apis/system.antrea.tanzu.vmware.com/v1beta1": context deadline exceeded
2021-12-22T01:42:03.78872321Z stderr F E1222 01:42:03.788656       1 available_controller.go:524] v1beta2.controlplane.antrea.io failed with: failing or missing response from https://10.96.128.183:443/apis/controlplane.antrea.io/v1beta2: Get "https://10.96.128.183:443/apis/controlplane.antrea.io/v1beta2": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
2021-12-22T01:42:03.788783882Z stderr F E1222 01:42:03.788717       1 available_controller.go:524] v1alpha1.stats.antrea.io failed with: failing or missing response from https://10.96.128.183:443/apis/stats.antrea.io/v1alpha1: Get "https://10.96.128.183:443/apis/stats.antrea.io/v1alpha1": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
2021-12-22T01:42:03.805540795Z stderr F E1222 01:42:03.805469       1 available_controller.go:524] v1beta1.system.antrea.tanzu.vmware.com failed with: Operation cannot be fulfilled on apiservices.apiregistration.k8s.io "v1beta1.system.antrea.tanzu.vmware.com": the object has been modified; please apply your changes to the latest version and try again
```

### CODE REVIEW

**Comment by yzaccc** [comment]
The ci failed because did not run "./hack/kind-fix-networking.sh", which is required to use antrea proxy on existing kind clusters, to use ovs.
refer:
`# This script is required for Antrea to work properly in a Kind cluster on Linux. It takes care of
# disabling TX hardware checksum offload for the veth interface (in the host's network namespace) of
# each Kind Node. This is required when using OVS in userspace mode. Refer to
# https://github.com/antrea-io/antrea/issues/14 for more information.`

File: .github/workflows/antrea-proxy.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          config: hack/kind-multi-worker-nocni.yaml
+          version: v0.11.0
+          node_image: kindest/node:v1.22.0@sha256:b8bda84bb3a190e6e028b1760d277454a72267a5454b57db34437c34a588d047
+      - name: Setup Antrea proxy in kind cluster
+        run: |
+          git clone https://github.com/antrea-io/antrea.git
+          cd antrea
+          git checkout tags/v1.4.0
+          kind get clusters
+          kind get nodes --name=chart-testing | xargs ./hack/kind-fix-networking.sh
```
Comment (knabben) [inline .github/workflows/antrea-proxy.yaml:29]: cool thanks

File: .github/workflows/antrea-proxy.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      - run: |
+          docker pull projects.registry.vmware.com/antrea/antrea-ubuntu:v1.4.0
+          kind load docker-image --name=chart-testing projects.registry.vmware.com/antrea/antrea-ubuntu:v1.4.0
+          kubectl apply -f https://github.com/antrea-io/antrea/releases/download/v1.4.0/antrea-kind.yml
+      - run: |
+          ./hack/install_metallb.sh
+      - run: |
+          go test -v ./tests
```
Comment (knabben) [inline .github/workflows/antrea-proxy.yaml:37]: can you replace by `make summary`

**Comment by knabben** [comment]
We could test with full Antrea-proxy capabilities and remove kube-proxy. For this we can

1) put the full configuration yaml (changed) on gist and use it.
2) update the configmap in the flight and restart antrea components.

**Review by yzaccc — APPROVED** [review]

**Pull request merged**